spiceaidocs/docs/data-connectors/graphql.md

```diff
@@ -265,6 +265,44 @@ sql> select node from stargazers limit 1;
 +------------------+--------+---------------+
 ```
 
-:::warning[Limitations]
-- Automatic object unnesting will overwrite columns if multiple properties contain the same sub-keys.
-:::
\ No newline at end of file
+#### Unnesting Duplicate Columns
+
+By default, the Spice Runtime will error when a duplicate column is detected during unnesting.
```

Comment:
Can you provide an example of the runtime error?

Reply:
Yes, providing example output would also be helpful.